src/lib.rs

```diff
@@ -136,7 +136,13 @@
 )]
 
 #[macro_use]
-extern crate html5ever;
+pub extern crate html5ever;
```

Comment:
Since this crate is on edition 2021 now, do we actually need the `#[macro_use]` here?

Reply:
hi @adamreichold  yeah, we still need for now.

Reply:
I meant that we actually added the necessary `use` clauses elsewhere but indeed it seems that `html5ever` does not properly scope its macros, so it would probably not really make things better.